src/traffic_logcat/Makefile.inc

```diff
@@ -47,7 +47,8 @@ traffic_logcat_traffic_logcat_LDADD = \
 	$(top_builddir)/iocore/utils/libinkutils.a \
 	$(top_builddir)/src/tscore/libtscore.la \
 	$(top_builddir)/src/tscpp/util/libtscpputil.la \
-        $(top_builddir)/mgmt/utils/libutils_p.la
+    $(top_builddir)/mgmt/utils/libutils_p.la \
+    $(top_builddir)/src/api/libtsapi.la
```

Comment:
nit: looks like inconsistent spacing. The previous lines are tabs, these new lines are spaced.

Reply:
I fixed these.